docs/source/api/serialization.rst

```diff
@@ -8,3 +8,5 @@ Serialization
 
    rustworkx.node_link_json
    rustworkx.read_graphml
+   rustworkx.parse_node_link_json_file
+   rustworkx.parse_node_link_json_str
```

Comment:
I really dislike the name of the functions. Maybe we can come up with alternatives? They feel ugly

Reply:
Yeah, I wasn't happy with them either tbh. I'll switch them to your suggested `parse_node_link_json` and `node_link_json_from_file` those are better names.

Reply:
I renamed the functions to: `parse_node_link_json` and `from_node_link_json_file` in: https://github.com/Qiskit/rustworkx/pull/1091/commits/52be0a0478de40e1c10ac463485adcb6a5ea78a9